### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by ashhath01** [deptofdefense/dds.mil#78]
*A/C*
consider accessibility

**Comment by marieksmith** [comment]
@ashhath01 did accessibility check on navigating (no mouse), and color contrast. sent Jeff the fixes. should be good for mvp.  marking as closed.
